packages/field/src/react/__stories__/tags-input-field.story.tsx

```diff
@@ -0,0 +1,250 @@
+import { Meta, Story } from '@storybook/react/types-6-0'
```

Comment:
What's the diff with tags-input-field and multi-select-field?  They seem essentially the same with just a bit of design diff.

Reply:
`TagsInput` does not use the combobox pattern where `MultiSelectField` does. TagsInput is do prove that we can support @KaidenR custom needs where a traditional combobox(`ActionMenu`) is not used.